urdf2webots/parserURDF.py

```diff
@@ -564,21 +564,21 @@ def getVisual(link, node, path):
                 visual.scale[2] = float(meshScale[2])
                 if visual.scale[0] * visual.scale[1] * visual.scale[2] < 0.0:
                     extension = os.path.splitext(meshfile)[1].lower()
-                    if extension in ['.dae', '.obj']:
+                    if extension in ['.dae', '.obj'] and targetVersion == 'R2022b':
```

Comment:
I believe it is better to test the other way round because features supported in R2022b will likely continue to work in upcoming versions (e.g., R2022c or R2023a, etc.).
```suggestion
                    if extension in ['.dae', '.obj'] and targetVersion != 'R2022a':
```

Reply:
but doesn't work retroactively for all existing versions, unless we put something like: `targetVersion not in ['R2022a', 'R2021b', 'R2021a', etc]`. Neither option seems especially nice actually

Reply:
might be better using comparators `< Version(2022, 1, 0` or something like it

Reply:
Relying on alphabetical order of the version string may be sufficient for the purpose of the comparison.

Reply:
```suggestion
                    if extension in ['.dae', '.obj'] and targetVersion >= 'R2022b':
```